src/customer-key-store/appsettings.json

```diff
@@ -19,16 +19,16 @@
   "JwtAudience": "https://aadrm.com",
   "JwtAuthorization": "https://login.windows.net/common/oauth2/authorize",
   "RoleAuthorizer": {
-    "LDAPPath": "If role authorization is used then this is the LDAP path"
+    "LDAPPath": "If you use role authorization (AuthorizedRoles) then this is the LDAP path."
   },
   "TestKeys": [
     { 
       "Name": "YourTestKeyName",
       "Id": "GUID",
-      "AuthorizedRoles": ["On prem AD groups that should have access to this key, if this value is present then the authorized emails will not be used"],
-      "AuthorizedEmailAddress": ["Emails of users that have access to this key. "],
-      "PublicPem" :  "The public key in PEM format.  It should not include the BEGIN and END lines",
-      "PrivatePem":  "The private key in PEM format.  It should not include the BEGIN and END lines"
+      "AuthorizedRoles": ["On premises Active Directory groups that you want to have access to this key. If you provide a value for AuthorizedRoles, then don't use AuthorizedEmailAddress."],

```

Comment:
>then don't use AuthorizedEmailAddress [](start = 149, length = 37)

then AuthorizedEmailAddress must be removed

Reply:
"AuthorizedRoles": ["On premises Active Directory groups that you want to have access to this key. If you provide a value for AuthorizedRoles, then remove the line that starts with AuthorizedEmailAddres."],
      "AuthorizedEmailAddress": ["Email addresses of users that have access to this key. If you provide a value for AuthorizedEmailAddress, then remove the line that starts with AuthorizedRoles."],